.gitignore

```diff
@@ -3,6 +3,8 @@ __pycache__/
 *.py[cod]
 *$py.class
 
+.idea/
```

Comment:
This is fine with me. But a potentially better approach would be to add it to your local ignore, so that it doesn't polute the gitignore (it is specific just to your machine)